server/index.js

```diff
@@ -1,16 +1,55 @@
-const express = require('express');
-const app = express();
-const bodyParser =  require('body-parser');
-const database = require('../database/controller.js')
-const PORT = 3050;
+var express = require('express');
+var bodyParser = require('body-parser');
+var database = require('../database');
+var app = express();
+app.use(bodyParser.json() );
 
-app.use(express.static('client/public'))
-app.use(bodyParser.json())
 
+// UNCOMMENT FOR REACT
+app.use(express.static(__dirname + '../client/public'));
 
-app.listen(PORT, () => {
-  console.log('Server is listening on port:', PORT);
+// UNCOMMENT FOR ANGULAR
+// app.use(express.static(__dirname + '/../angular-client'));
+// app.use(express.static(__dirname + '/../node_modules'));
+
+// app.get('/api/users', function (req, res) {
+//   db.getAllUsers(req.body.name, req.body.id, req.body.balance, res, (err,data) =>{
+//     if(err){
+//       res.status(500);
+//       res.send(err);
+//     } else{
+//       res.status(200);
+//       res.send(data)
+//     }
+//   })
+
+// });
+
+
+app.get('/api/tesla/suggestions', (req, res) => {
+  database.getAll(res);
 })
```

Comment:
This endpoint looks hard-coded, but also like it's getting all the data when it only should retrieve a single row.